.env.example

```
@@ -146,6 +146,10 @@ NEXT_PUBLIC_E2E_MODE=
 NEXT_PUBLIC_BUILD_SHA=
 # Public URL of the demo video embedded on the landing page.
 DEMO_VIDEO_URL=
+# Public URL of the pay-flow video shown on the /artist-profiles section.
+# If unset, the page falls back to the static monetization section. Host on
+# Vercel Blob (*.blob.vercel-storage.com - already CSP-allowlisted).
+ARTIST_PROFILES_PAY_FLOW_VIDEO_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ARTIST_PROFILES_PAY_FLOW_VIDEO_URL key should go before the DEMO_VIDEO_URL key

(UnorderedKey)